Letterbook.Workers/Consumers/TimelineConsumer.cs

```diff
@@ -33,30 +34,30 @@ public async Task Consume(ConsumeContext<PostEvent> context)
 				case nameof(PostEventPublisher.Created):
 					return;
 				case nameof(PostEventPublisher.Deleted):
-					await _timelines.As(context.Message.Claims).HandleDelete(post);
+					await _timelines.As(context.Message.Claims.Select(c => (Claim)c)).HandleDelete(post);
```

Comment:
this Select statement gets repeated a lot. Could it be worth factoring out a helper method or something?